cfme/js.py

```diff
@@ -7,7 +7,7 @@
 
 return {
     jquery: jQuery.active,
-    prototype: Ajax.activeRequestCount,
+    prototype: (typeof Ajax == "undefined") ? 0 : Ajax.activeRequestCount,
```

Comment:
I'd rather avoid the potential for weird JS bugs, so using the no-really-are-they-equal operator seems prudent (`===` vs. `==`).